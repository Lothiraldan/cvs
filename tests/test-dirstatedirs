#!/bin/sh

# test dirstate._dirs refcounting
hg init t
cd t
mkdir -p a/b/c/d
touch a/b/c/d/x
touch a/b/c/d/y
touch a/b/c/d/z
hg ci -Am m
hg mv a z
