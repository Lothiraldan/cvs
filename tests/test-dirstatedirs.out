adding a/b/c/d/x
adding a/b/c/d/y
adding a/b/c/d/z
moving a/b/c/d/x to z/b/c/d/x
moving a/b/c/d/y to z/b/c/d/y
moving a/b/c/d/z to z/b/c/d/z
